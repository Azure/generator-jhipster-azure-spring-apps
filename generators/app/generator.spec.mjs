import { expect } from 'expect';

import { helpers, lookups } from '#test-utils';

const SUB_GENERATOR = 'app';
const BLUEPRINT_NAMESPACE = `jhipster:${SUB_GENERATOR}`;

describe('SubGenerator app of azure-spring-apps JHipster blueprint', () => {
  describe('run', () => {
    let result;
    before(async function () {
      result = await helpers
        .create(BLUEPRINT_NAMESPACE)
        .withOptions({
          reproducible: true,
          defaults: true,
          blueprint: 'azure-spring-apps',
        })
        .withLookups(lookups)
        .run();
    });

    it('should succeed', () => {
      expect(result.getStateSnapshot()).toMatchSnapshot();
    });
  });
});
