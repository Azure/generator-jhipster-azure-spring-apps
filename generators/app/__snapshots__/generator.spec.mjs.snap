// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubGenerator app of azure-spring-apps JHipster blueprint run should succeed 1`] = `
Object {
  ".devcontainer/Dockerfile": Object {
    "stateCleared": "modified",
  },
  ".devcontainer/devcontainer.json": Object {
    "stateCleared": "modified",
  },
  ".gitattributes": Object {
    "stateCleared": "modified",
  },
  ".github/CODE_OF_CONDUCT.md": Object {
    "stateCleared": "modified",
  },
  ".github/ISSUE_TEMPLATE.md": Object {
    "stateCleared": "modified",
  },
  ".github/PULL_REQUEST_TEMPLATE.md": Object {
    "stateCleared": "modified",
  },
  ".github/workflows/build-with-maven.yml": Object {
    "stateCleared": "modified",
  },
  ".gitignore": Object {
    "stateCleared": "modified",
  },
  ".mvn/wrapper/maven-wrapper.jar": Object {
    "stateCleared": "modified",
  },
  ".mvn/wrapper/maven-wrapper.properties": Object {
    "stateCleared": "modified",
  },
  ".yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "CHANGELOG.md": Object {
    "stateCleared": "modified",
  },
  "CONTRIBUTING.md": Object {
    "stateCleared": "modified",
  },
  "LICENSE": Object {
    "stateCleared": "modified",
  },
  "LICENSE.md": Object {
    "stateCleared": "modified",
  },
  "NOTICE.txt": Object {
    "stateCleared": "modified",
  },
  "README.md": Object {
    "stateCleared": "modified",
  },
  "assets/web.png": Object {
    "stateCleared": "modified",
  },
  "client/Dockerfile": Object {
    "stateCleared": "modified",
  },
  "client/README.md": Object {
    "stateCleared": "modified",
  },
  "client/entrypoint.js": Object {
    "stateCleared": "modified",
  },
  "client/entrypoint.sh": Object {
    "stateCleared": "modified",
  },
  "client/nginx/nginx.conf": Object {
    "stateCleared": "modified",
  },
  "client/package-lock.json": Object {
    "stateCleared": "modified",
  },
  "client/package.json": Object {
    "stateCleared": "modified",
  },
  "client/pom.xml": Object {
    "stateCleared": "modified",
  },
  "client/public/favicon.ico": Object {
    "stateCleared": "modified",
  },
  "client/public/index.html": Object {
    "stateCleared": "modified",
  },
  "client/public/manifest.json": Object {
    "stateCleared": "modified",
  },
  "client/public/web.config": Object {
    "stateCleared": "modified",
  },
  "client/src/@types/window.d.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/App.css": Object {
    "stateCleared": "modified",
  },
  "client/src/App.test.tsx": Object {
    "stateCleared": "modified",
  },
  "client/src/App.tsx": Object {
    "stateCleared": "modified",
  },
  "client/src/actions/actionCreators.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/actions/common.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/actions/itemActions.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/actions/listActions.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/components/telemetry.tsx": Object {
    "stateCleared": "modified",
  },
  "client/src/components/telemetryContext.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/components/todoContext.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/components/todoItemDetailPane.tsx": Object {
    "stateCleared": "modified",
  },
  "client/src/components/todoItemListPane.tsx": Object {
    "stateCleared": "modified",
  },
  "client/src/components/todoListMenu.tsx": Object {
    "stateCleared": "modified",
  },
  "client/src/config/index.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/index.css": Object {
    "stateCleared": "modified",
  },
  "client/src/index.tsx": Object {
    "stateCleared": "modified",
  },
  "client/src/layout/header.tsx": Object {
    "stateCleared": "modified",
  },
  "client/src/layout/layout.tsx": Object {
    "stateCleared": "modified",
  },
  "client/src/layout/sidebar.tsx": Object {
    "stateCleared": "modified",
  },
  "client/src/models/applicationState.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/models/index.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/models/todoItem.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/models/todoList.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/pages/homePage.tsx": Object {
    "stateCleared": "modified",
  },
  "client/src/react-app-env.d.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/reducers/index.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/reducers/listsReducer.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/reducers/selectedItemReducer.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/reducers/selectedListReducer.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/reportWebVitals.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/services/itemService.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/services/listService.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/services/restService.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/services/telemetryService.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/setupTests.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/ux/styles.ts": Object {
    "stateCleared": "modified",
  },
  "client/src/ux/theme.ts": Object {
    "stateCleared": "modified",
  },
  "client/tsconfig.json": Object {
    "stateCleared": "modified",
  },
  "mvnw": Object {
    "stateCleared": "modified",
  },
  "mvnw.cmd": Object {
    "stateCleared": "modified",
  },
  "pom.xml": Object {
    "stateCleared": "modified",
  },
  "web/README.md": Object {
    "stateCleared": "modified",
  },
  "web/pom.xml": Object {
    "stateCleared": "modified",
  },
  "web/src/main/java/com/mycompany/myapp/SimpleTodoApplication.java": Object {
    "stateCleared": "modified",
  },
  "web/src/main/java/com/mycompany/myapp/configuration/RFC3339DateFormat.java": Object {
    "stateCleared": "modified",
  },
  "web/src/main/java/com/mycompany/myapp/configuration/WebConfiguration.java": Object {
    "stateCleared": "modified",
  },
  "web/src/main/java/com/mycompany/myapp/model/TodoItem.java": Object {
    "stateCleared": "modified",
  },
  "web/src/main/java/com/mycompany/myapp/model/TodoList.java": Object {
    "stateCleared": "modified",
  },
  "web/src/main/java/com/mycompany/myapp/model/TodoState.java": Object {
    "stateCleared": "modified",
  },
  "web/src/main/java/com/mycompany/myapp/repository/TodoItemRepository.java": Object {
    "stateCleared": "modified",
  },
  "web/src/main/java/com/mycompany/myapp/repository/TodoListRepository.java": Object {
    "stateCleared": "modified",
  },
  "web/src/main/java/com/mycompany/myapp/web/HomeController.java": Object {
    "stateCleared": "modified",
  },
  "web/src/main/java/com/mycompany/myapp/web/TodoListsController.java": Object {
    "stateCleared": "modified",
  },
  "web/src/main/resources/application.yml": Object {
    "stateCleared": "modified",
  },
}
`;
